tasks/Debian-tarball.yml

```diff
@@ -0,0 +1,12 @@
+---
+- include: tarball.yml
+
+- include: upstart.yml
+  tags: deploy
+
+- include: systemd.yml
+  tags: deploy
```

Comment:
Check 15.04

```bash
vagrant@default-ubuntu-1504:~$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 15.04
Release:	15.04
Codename:	vivid
vagrant@default-ubuntu-1504:~$ ls -la /usr/lib | grep systemd
drwxr-xr-x  7 root root    4096 Mar 29 21:45 systemd
vagrant@default-ubuntu-1504:~$ ls -la /etc | grep init
drwxr-xr-x  2 root root    4096 Mar 29 21:50 init
drwxr-xr-x  2 root root    4096 Mar 29 21:52 init.d
drwxr-xr-x  5 root root    4096 Mar 29 21:42 initramfs-tools
```

it contains both init and systemd which might lead to unexpected behavior 


15.10 yields the same

```bash
vagrant@vagrant:~$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 15.10
Release:	15.10
Codename:	wily
vagrant@vagrant:~$ ls -la /etc/ | grep init
drwxr-xr-x  2 root root    4096 Aug  1 16:27 init
drwxr-xr-x  2 root root    4096 Aug  1 16:28 init.d
drwxr-xr-x  5 root root    4096 Aug  1 16:22 initramfs-tools
vagrant@vagrant:~$ ls -la /usr/lib/ | grep systemd
drwxr-xr-x  8 root root    4096 Aug  1 16:24 systemd
```